packages/expo-gl/src/GLView.web.js

```diff
@@ -1,14 +1,174 @@
 // @flow
+import { UnavailabilityError } from 'expo-errors';
+import PropTypes from 'prop-types';
+import * as React from 'react';
+import { Dimensions, StyleSheet } from 'react-native';
 
-import React from 'react';
-import { Text, View } from 'react-native';
+import type { SnapshotOptions } from './GLView.types';
+
+type Props = {
+  /**
+   * Called when the OpenGL context is created, with the context object as a parameter. The context
+   * object has an API mirroring WebGL's WebGLRenderingContext.
+   */
+  onContextCreate?: (gl: *) => void,
+};
+
+function _getImageForAsset(asset) {
+  if (asset == null) return null;
```

Comment:
write out the curly braces, we do it almost everywhere else

```suggestion
  if (asset == null) {
    return null;
  }
```